cf/service_broker_test.go

```diff
@@ -184,22 +206,7 @@ var _ = Describe("Client ServiceBroker", func() {
 
 		Context("when a broker with the specified name exists in CC", func() {
 			BeforeEach(func() {
-				ccResponse = cfclient.ServiceBrokerResponse{
-					Count: 1,
-					Pages: 1,
-					Resources: []cfclient.ServiceBrokerResource{
-						{
-							Meta: cfclient.Meta{
-								Guid: testBroker.GUID,
-							},
-							Entity: cfclient.ServiceBroker{
-								Name:      brokerName,
-								BrokerURL: testBroker.BrokerURL,
-								Username:  brokerUsername,
-							},
-						},
-					},
-				}
+				ccResponse = ccBrokersResponse(ccGlobalBroker)
```

Comment:
better move this inside the `It` as it does apply to the next test case for the space-scoped broker